device-types/HPE/5130-48G-PoE+-4SFP+.yml

```diff
@@ -0,0 +1,177 @@
+manufacturer: HPE
+model: 5130-48G-PoE+-4SFP+ (370W) EI
+slug: 5130-48G
+part_number: JG937A
+u_height: 1
+is_full_depth: false
+comments: "https://h20195.www2.hpe.com/v2/GetPDF.aspx/c04394228.pdf\r\n17.32(w) x\
+  \ 14.17(d) x 1.72(h) in.\r\n(44 x 36 x 4.36 cm) (1U height)\r\n17.64 lb (8 kg)\r\
+  \n\r\nmax power rating 490 W\r\n\r\n160/1671 BTU/hr (168.8/1762.91 kJ/hr), for AC\
+  \ power. For DC power minimum heat dissipation is 147 BTU/hr and 3037 BTU/hr maximum."
```

Comment:
Your comment does not import at all due to the formatting.  I tested the import on [netboxdemo.com](https://netboxdemo.com) (user / pass: netbox /netbox) .  You can always write the comment in any netbox webgui and export the device type from the gui to see proper formatting.

Reply:
Confusingly this was an export from my netbox install. Perhaps I was too ambitious with it as others may not want all the detail. I think a link to a datasheet is important though.

I'll edit and re-commit

Reply:
I am new so probably wrong, but I'm unable to import any comment into the demo site.

If I edit a device type and add a single word test123 to the comments section it saves and displays as normal. If I export that I get the line in YAML:

> comments: test123

If I import YAML with that line the comment field is blank. I can't find any way to import a device-type as a YAML file without having a blank comment field. No error messages anywhere.

Reply:
After I replied I started thinking it could be a bug with comments and importing.

For a first time contributor that was a little cautious, you really followed through on this.  

Thanks!

You can troubleshoot some more to see if a possible bug report to the NetBox core repo is needed.  If not, I’m happy to do so also.

Reply:
I've proven this to be a bug with a very simple device type definition. The following imports fine but the comments are lost.


```
manufacturer: Cisco
model: Switch_with_comments
slug: testSwitch
u_height: 1
is_full_depth: false
comments: disappear 
interfaces:
- name: ge1/0/0
  type: 1000base-t
  mgmt_only: false
```
  

I don't have access to other versions (mine is the same as the demo) so don't know when this started. I'll attempt a bug report. 

In the meantime I've committed an updated device model with no comments and improved interface names as per the contribution guidelines.